civihr_default_theme/scss/pages/_onboarding-wizard.scss

```diff
@@ -1,4 +1,5 @@
 $carousel-box-shadow-color: rgba(#3f3c55,0.3);
+$crm-darker-grayblue: rgb(182, 186, 207);
```

Comment:
I can see this variable was used in the file "./civihr_default_theme/scss/pages/_onboarding-wizard.scss" - how did it work at all before without its definition?

Also, should not it go to `civihr_default_theme/scss/base/_variables.scss`?

Reply:
well, I have fixed it, wrong scss variable `$crm-darker-grayblue` was used.